types/knex/knex-tests.ts

```diff
@@ -1111,6 +1111,7 @@ knex('users')
 //
 // Migrations
 //
+const name = "test";
```

Comment:
???

Reply:
The test otherwise to refers to the global `name`, which the linter doesn't like.